src/01-dinosaur-facts.js

```diff
@@ -22,7 +22,24 @@ const exampleDinosaurData = require("../data/dinosaurs");
  *  getLongestDinosaur(dinosaurs);
  *  //> { Brachiosaurus: 98.43 }
  */
-function getLongestDinosaur(dinosaurs) {}
+function getLongestDinosaur(dinosaurs) {
+  if(!dinosaurs.length){
```

Comment:
This conditional is unnecessary. Since you initialized `biggestDinosaur` with an empty object, if the dino array is empty is will return the empty object on line 40.